stock_picking_sequence/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Stock Picking Sequence
+======================
+
+This module extends the functionality of report to This module adds the sequence and total quantity into the delivery order printout.
```

Comment:
This module extends the functionality of report to adds the sequence and total quantity into the delivery order printout.